### ISSUE TITLE
'BP5ConnectNoti' is never received

### ISSUE THREAD

**Issue by miguelrodlav** [iHealthDeviceLabs/iHealthDeviceLabs-iOS#13]
Hi,

I'm doing testings with the BP5 and the BP7 devices

I can pair the devices properly through iPhone Bluetooth settings, and the official iHealth App connect and measure just fine with both devices.

But when I'm running the iHealth iOS SDK Test app (right after cloning the repository), the BP5 is never recognized by it.

The app setup a Notification Observer with:

```
[[ NSNotificationCenter defaultCenter ] addObserver : self selector :@selector (DeviceConnectForBP5:) name : BP5ConnectNoti object : nil];
[[ NSNotificationCenter defaultCenter ] addObserver : self selector :@selector (DeviceDisConnectForBP5:) name : BP5DisConnectNotiobject : nil ];
```
This notification is never received, so the selector never triggers. (For the equivalents observers added for BP7, the notification is received and the selector executes fine).

If I try to run the selector manually:
```
BP5Controller *controller = [ BP5Controller shareBP5Controller ];
NSArray *bpDeviceArray = [controller getAllCurrentBP5Instace ];
```
the bpDeviceArray always comes empty.

In short, BP7 works fine, but BP5 never gets recognized.
Any idea why is this happening?
Thanks.

**Comment by Realank** [comment]
Hi miguelrodlav,

Thanks for your information, could you please show me your SN number of BG5?
At bottom of the BG5, near the cuff, looks like: SN 2015-xxxxxx

**Comment by miguelrodlav** [comment]
Hi Realank, 

It's a BP5 device (Blood Pressure), not BG5.
The SN is 2016-160700015171008.

I'm really desperate here cause I needed the implementation for last week... And I don't know what is going on...

Thank you very much!

**Comment by Realank** [comment]
Hi, Sorry for wrong tying, it's BP5

I also reproduce your issue with SN2016 BP5, I will find out the bug and fix it, 
Thanks you and sorry for inconvenience

**Comment by Realank** [comment]
Hi miguelrodlav,

I found out the bug, hear this the solution:
###Please add a protocol in plist file:
- 1. open iHealthDemoCode-Info.plist file
- 2. unfold field: Supported external accessory protocols
- 3. add a new item, and the value is "com.jiuan.BPV23"
- 4. re-run your app, it will work

Hopefully it's helpful for you

**Comment by miguelrodlav** [comment]
It's working now!!
Thanks!!!

**Comment by Realank** [comment]
Great!
Thanks

**Comment by sivachandiran** [comment]
Hi, 
i need iHealth SDK for integrate my iPhone native project, i search but i didnt get SDK . Can you please provide SDK.

Thanks,
